tests/NugetUtility.Tests/MethodsTests.cs

```diff
@@ -224,7 +226,7 @@ public async Task GetPackages_AllowedLicenses_Should_Throw_On_MIT()
 
             result.Should().HaveCount(1);
             validationResult.IsValid.Should().BeFalse();
-            validationResult.InvalidPackages.Count.Should().Be(4);
+            validationResult.InvalidPackages.Count.Should().Be(5);

```

Comment:
Why the previous test fails now? I am not sure that I understand what changed for changing the counters?

Reply:
Yes. One more package in the project, one more package in the test.